src/components/shared/Entity/DetailsForm.tsx

```diff
@@ -93,6 +93,12 @@ function DetailsForm({ connectorTags, messagePrefix }: Props) {
                     pattern: `^([a-zA-Z0-9-_.]+/)+[a-zA-Z0-9-_.]+$`,
                     type: 'string',
                 },
+                description: {
+                    description: intl.formatMessage({
+                        id: 'description.description',
+                    }),
+                    type: 'string',
+                },
```

Comment:
Adding description to the details form. I named it `description` only because the field is titled `details` in the DB and it felt weird have `details.details` in the UI